app/client/src/pages/Editor/CustomWidgetBuilder/Editor/ChatBot/ChatBot.tsx

```
@@ -0,0 +1,97 @@
+} from "../../constants";
+import { isObject } from "lodash";
+
+export const ChatBot = (props: ContentProps) => {
+  const ref = useRef<HTMLIFrameElement>(null);
+  const { parentEntityId, uncompiledSrcDoc, update, widgetId } = useContext(
+    CustomWidgetBuilderContext,
+  );
+
+  const handleSrcDocUpdates = useCallback(() => {
+    // Send src doc to the chatbot iframe
+    if (ref.current && ref.current.contentWindow && uncompiledSrcDoc) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)